plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -55,21 +55,12 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 			if ( i > 0 ) {
 				await page.click( 'button.add_attribute' );
 			}
+			await page.waitForSelector( `input[name="attribute_names[${ i }]"]` );
 
-			await page
-				.locator(
-					`.woocommerce_attribute_data input[name="attribute_names[${ i }]"]`
-				)
-				.fill( `attr #${ i + 1 }` );
-			await page
-				.locator(
-					`.woocommerce_attribute_data textarea[name="attribute_values[${ i }]"]`
-				)
-				.fill( 'val1 | val2' );
+			await page.locator( `input[name="attribute_names[${ i }]"]` ).first().type( `attr #${ i + 1 }` );
+			await page.locator( `textarea[name="attribute_values[${ i }]"]` ).first().type( 'val1 | val2' );
```

Comment:
Simplified this based on @rodelgc's feedback.  There is a JS event that fires, so we'll use .type() instead of .fill() for these fields.  Additionally, this was erroring out because there were two attribute_names and attribute_values fields... so we're just forcing it to use the first one now.